docs/hard-fork-upgrade/prerequisites-and-planning.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Archive migration prerequisites
+sidebar_label: Archive migration prerequisites
+hide_title: true
+description: Mina protocol havily relies on its archive node. It is crucial to properly migrate archives from Mainnet to Berkeley.
+keywords:
+  - hard fork
+  - upgrade
+  - archive migration
+  - planning 
+  - prerequisites
+  - mina archive node
+  - archive node
+---
+
+# Migrating Mainnet Database to Berkeley
+
+Before you start planning the migration of your Mainnet database into the Berkeley version of the Mina network, read this document in entirety.
+
+## Overview
+
+Mainnet database requires to be migrated in two steps. Each step uses an application that is shipped by Mina Foundation and o1Labs teams:
```

Comment:
```suggestion
Before you start planning the migration of your archive node database into the Berkeley version of the Mina network, read this document in entirety.

## Overview

The archive node database requires to be migrated in two steps. Each step uses an application that is shipped by Mina Foundation and o1Labs teams:
```

Reply:
@dkijania  can you verify for me that I didn't get this wrong? thankies

Reply:
@barriebyron, yes. there are ok :)

Reply:
@barriebyron, yes. there are ok :)